messaging-client-databus/src/main/java/com/inmobi/messaging/consumer/databus/DatabusConsumerConfig.java

```diff
@@ -33,4 +33,6 @@
   public static final String waitTimeForFlushConfig =
   "databus.consumer.waittime.forcollectorflush";
   public static final long DEFAULT_WAIT_TIME_FOR_FLUSH = 5000; // 5 second
+  public static final String frequencyForDiscoverer =  "databus.consumer.collector.discoverer.frequency.seconds"; // in seconds
+  public static final int  DEFAULT_FREQUENCY_FOR_DISCOVERER = 120; // 2 minutes
```

Comment:
Can we increase this *default value* as adding a new collector or increasing the number of threads per collector is not too often.